locksmith/src/operations/authorizedLockOperations.ts

```diff
@@ -11,7 +11,7 @@ export const hasAuthorization = async (
   const lockAddress = Normalizer.ethereumAddress(address)
   const web3Service = new Web3Service(networks)
 
-  const { wallet } = await getPurchaser({ network })
+  const wallet = await getPurchaser({ network })
```

Comment:
it seems the function still return `{ wallet, provider}` so why remove the brackets?